src/ViewRenderer.php

```diff
@@ -498,7 +500,13 @@ private function findLayoutFile(string $file, WebView $view): string
             return $file;
         }
 
-        return $file . '.' . $view->getDefaultExtension();
+        $layoutFile = $file . '.' . $view->getDefaultExtension();
+
+        if ($view->getDefaultExtension() !== 'php' && !is_file($layoutFile)) {
+            $layoutFile = $file . '.php';
+        }
```

Comment:
Why need it?

Reply:
It's needed when layout is php template and other views is different template e.g twig. See https://github.com/yiisoft/view/blob/f378b696917c2a086b634ef072ccd19b16859f88/src/ViewTrait.php#L615-L619